_posts/2021-12-13-log4shell-does-not-affect-trino.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Log4Shell does not affect Trino"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+In the last few days we had a surge of folks in our community reaching out with
+concerns over the [Log4Shell exploit](https://www.lunasec.io/docs/blog/log4j-zero-day/)
+([CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228)),
+and we wanted to inform you that Trino is not affected. Trino does not use log4j
+in the core engine or runtime classes. There are some connectors that include 
+the log4j dependency from client dependencies, but some are not used or are not
+versions affected by the Log4Shell vulnerability. Rest assured, we are taking 
+extra steps to make sure no future iterations of Trino will ever introduce this 
```

Comment:
Dont say it like that .. just mention that regular security reviews, including code and dependency analysis, are part of the regular development process. And if there are any findings from the community - how to reach out (but that part probably later in the post)